docs/reference/index-modules/similarity.asciidoc

```diff
@@ -168,7 +168,7 @@ Type name: `LMJelinekMercer`
 ==== Default and Base Similarities
 
 By default, Elasticsearch will use whatever similarity is configured as
-`default`. However, the similarity functions `queryNorm()` and `coord()`
+`default` or `BM25` if default is not declared. However, the similarity functions `queryNorm()` and `coord()`
```

Comment:
> Even though we currently allow to redefine the BM25 similarity,

Not anymore: https://github.com/elastic/elasticsearch/pull/16682

I am trying to say that the default is `BM25` but I'll do differently.  The `bm25`explanation should be first and declared as the default for ES.